nightwatch.conf.js

```diff
@@ -15,7 +15,7 @@ require('nightwatch-cucumber')({
 module.exports = {
   output_folder: 'reports',
   custom_assertions_path: '',
-  page_objects_path: ['page_objects', 'page_objects/florence', 'page_objects/web'],
+  page_objects_path: ['page_objects', 'page_objects/geography', 'page_objects/web', 'page_objects/florence'],
```

Comment:
I don't think we'll need the geography page object in it's own folder, it should be able to live under `web`.

Reply:
web tests is broken